src/main/java/com/powsybl/openloadflow/ac/AcloadFlowEngine.java

```diff
@@ -195,7 +192,7 @@ public AcLoadFlowResult run() {
     }
 
     public static <T> List<AcLoadFlowResult> run(T network, LfNetworkLoader<T> networkLoader, AcLoadFlowParameters parameters, Reporter reporter) {
-        return LfNetwork.load(network, networkLoader, parameters.getNetworkParameters(), reporter)
+        return LfNetwork.load(network, networkLoader, new ArrayList<>(), new ArrayList<>(), parameters.getNetworkParameters(), reporter)
```

Comment:
You must do a load method without this two lists because mort of the time we don't need it.